qiwis.py

```diff
@@ -164,7 +164,7 @@ def __init__(
             parent: A parent object.
         """
         super().__init__(parent=parent)
-        self.appInfos = appInfos
+        self.appInfos: Dict[str, AppInfo] = {}

```

Comment:
We didn't write down any attributes in `Qiwis` class strangely.
I think we need to do this not in this RP, but later.

Reply:
You mean the attribute docstring? Yeah 😄

Reply:
Yes👍